UseCase_I.ipynb

```diff
@@ -109,7 +109,7 @@
    "outputs": [],
    "source": [
     "# Let's see whether we are dealing with strings or byte strings\n",
-    "objects_in_paper[0]['Type']"
+    "print(type(objects_in_paper[0]['Type']))"
```

Comment:
Byte strings have not been an issue since the astropy 4.1 release. It is recommended to remove this cell and the next one to address #74 as these are no longer needed.